store/store.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strconv"
 
 	"github.com/gogo/protobuf/proto"
+	format "github.com/ipfs/go-ipld-format"
```

Comment:
Let's use `ipld` alias for `go-ipld-format` everywhere. I also did that in the previous PR

Reply:
changed here https://github.com/lazyledger/lazyledger-core/pull/356/commits/b2469cccbfed2e68d07bfe765a8a69cf9373cc96

left it as `format` in the `consensus` and `light` packages as they use our own `ipld` package

Reply:
Yep, I am going to propose the rename of `ipld` package soon. We should change its name as it does not explain what it has inside